fem/hybridization.hpp

```diff
@@ -64,6 +64,11 @@ class Hybridization
    FiniteElementSpace *fes, *c_fes;
    BilinearFormIntegrator *c_bfi;
 
+   /// Set of constraint boundary face integrators to be applied.
+   Array<BilinearFormIntegrator*> c_bfbfi;
```

Comment:
I realize there is a precedent in MFEM for using acronyms as variables names, but you could start a trend here of using more descriptive variable names.

Reply:
Actually, `BilinearForm` now uses more descriptive names, which I support, I just wanted to follow `c_bfi` above. Renamed 👍 .